src/docs/components/ComponentReference/SlotsReference.tsx

```diff
@@ -34,20 +35,22 @@ export class SlotsReference extends TsxComponent<Props> {
   public render() {
     const rowSlot: ScopedRowSlot<SlotItem> = ({item}) => {
       return (
-        <TableRow slot='row'>
-          <TableCell>
-            {item.name === '' ?
-              <span style='color: rgb(200, 200, 200);'>default</span> :
-              <span>{item.name}</span>
-            }
-          </TableCell>
-          <TableCell>{item.description}</TableCell>
-        </TableRow>
+        <row-template slot='row'>
+          <TableRow>
+            <TableCell>
+              {item.name === '' ?
+                <span style='color: rgb(200, 200, 200);'>default</span> :
```

Comment:
color css assigment without a class. Hard to override

Reply:
This is part of the documentation and not a component intended to be used by anyone except us.